config/application.rb

```diff
@@ -76,6 +76,9 @@ class Application < Rails::Application
     config.external_url_mapping = APP_CONFIG = YAML.load(File.read(Rails.root + "config/external_link_mapping.yml"))
 
     config.before_initialize do
+      Settings.add_source!(Rails.root.join('config', 'hets.yml').to_s)
```

Comment:
Have you tested it? On my machine the `settings.local.yml` has overwritten the preferences set in `hets.yml`.